content/sensu-go/5.12/reference/agent.md

```diff
@@ -1253,6 +1255,41 @@ example               | {{< highlight shell >}}"enable_env": true{{< /highlight
 
 {{< /language-toggle >}}
 
+### Configuration via environment variables
+
+The `sensu-agent` service configured by our supported packages will read environment variables from `/etc/default/sensu-agent` on Debian/Ubuntu systems and `/etc/sysconfig/sensu-agent` on RHEL systems. These files are not created by the installation package, so you will need to create them.
+
+{{< language-toggle >}}
+
+{{< highlight "Ubuntu/Debian" >}}
+$ sudo touch /etc/default/sensu-agent
+{{< /highlight >}}
+
+{{< highlight "RHEL/CentOS" >}}
+$ sudo touch /etc/sysconfig/sensu-agent
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+For any configuration flag you wish to specify as an environment variable, you will need to append `SENSU_` and convert any dashes (`-`) to underscores (`_`). Then, add the resulting environment variable to the appropriate environment file described above. You must restart the service for these settings to take effect.
+
+In the following example, the `API-HOST` flag is configured as an environment variable and set to `"0.0.0.0"`.
```

Comment:
```suggestion
In the following example, the `api-host` flag is configured as an environment variable and set to `"0.0.0.0"`.
```

`api-host` is the parameter name we'd use in agent.yml or as `--api-host` on the command line.